plugins/woocommerce/client/blocks/assets/js/blocks/checkout/address-wrapper/index.tsx

```diff
@@ -16,15 +16,18 @@ export const AddressWrapper = ( {
 	isEditing = false,
 	addressCard,
 	addressForm,
+	shouldAnimate = false,
```

Comment:
Temporary fix. The address form gets collapsed if only address_1 and names are filled in. On first render it opens with an animation. I am disabling the animation until this is fixed in https://github.com/woocommerce/woocommerce/issues/58622